### PR TITLE
docs: start presenting the project

### PR DESCRIPTION
**Pull request by rehia**
first version of a presentation document, to facilitate newcomers integration

### CODE REVIEW

File: join-us.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Presentation
+
+**_Design all the things_** is an open initiative imagined by [Sylvain Fraïssé](https://github.com/fraisse) and [Jérôme Avoustin](https://github.com/rehia) to explore and learn how a "true" _Domain Driven Design_ approach could help us write better softwares. The purpose is not to build a product, but to learn new things, and make our mental models evolve.
+
+It is organized around face-to-face workshops, and probably remote as well. They are open to anyone who wants to contribute. You can come and contribute to the whole project, or just come once. We will learn together. If you want to know more, join the [_Montpellier Communautés_ slack](https://bit.ly/comm-mtp), and follow the #ddd channel. We organize meetups on it.
```
Comment (Sy1v4in) [inline join-us.md]: > and follow the #ddd channel

-> and follow the `#ddd` channel (`design all the things` keyword).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # releasy
+
 Web app to follow release workflow, from ticket creation to production
```
Comment (Sy1v4in) [inline README.md]: > ticket creation

is not in the ubiquitous language 😛

File: join-us.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## The Product
+
+Because we can't follow a _Domain Driven Design_ approach if we don't have at least a problem to solve, we came with one which presents at least 3 main interests:
+
+1. It's a problem we, as developers, meet every day, and we have gathered some knowledge about, making us legitimate product owners
+2. We will use the solution to build it, and _eat our own dog food_
+3. The problem is complex enough, and the domain language rich enough to justify such an approach.
+
+_**Releasy**_ is a system indended to help product owners/managers, developers and other team members give a better transparency on _how _[the] _project is coming along_, and try to better understand how their build workflow is going on, from the feature declaration to production.
```
Comment (Sy1v4in) [inline join-us.md]: > indended 

-> intended ?

File: README.md (hunk 2 above)
Comment (jerome-fl) [reply README.md]: it was written before hand :p

**Comment by jerome-fl** [comment]
@fraisse fixed

**Review by Sy1v4in — APPROVED** [review]

**Comment by Sy1v4in** [comment]
👌

**Pull request merged**